src/node/historical_queries_adapter.cpp

```diff
@@ -509,6 +509,28 @@ namespace ccf::historical
         }
       }
 
+      // If recovery in progress, prohibit any historical queries for previous
+      // epochs.
```

Comment:
```suggestion
      // epochs, because the service does not yet have access to the
      // ledger secrets necessary to produce commit evidence.
```